facerecognition.py

```diff
@@ -14,6 +14,7 @@ def __init__(self, height, width):
         self.y = random.randrange(0, self.height, 1)
         self.color = (0, 255, 0)
         self.danger = False
+        self.useTargeting = random.randrange(1,3,1)

```

Comment:
What do you think about making this variable a boolean instead of an integer?

Reply:
Nice catch! Thanks will do.